src/Pretty.hs

```diff
@@ -608,7 +619,8 @@ instance Pretty VarInfo (ExprF Syntax) where
     pure $ parens (group (vsep ppXs))
 
 instance Pretty VarInfo Closure where
-  pp _ _ = pure $ text "#<closure>"
+  pp _ (FO fo) = pure $ "#<" <> text (_stxValue (_closureIdent fo)) <> ">"
+  pp _ (HO n _) = pure $ "#<" <> text n <> ">"
```

Comment:
FYI here are the changes to the pretty printer that print the closure name rather than the type "closure". In some cases this seems desirable, but the more I look at it having "foo<closure>" also seems reasonable. Thoughts?